modules/transport/PostMessageTransportBackend.js

```diff
@@ -43,7 +43,8 @@ const LEGACY_OUTGOING_METHODS = [
     'participant-left',
     'video-conference-joined',
     'video-conference-left',
-    'video-ready-to-close'
+    'video-ready-to-close',
+    'report-a-problem-pressed'
```

Comment:
We don't want to include new events here. The purpose of the array is only for backward compatibility for older events that were used by the embedder of Jitsi Meet without using the external api.